README.md

```diff
@@ -1,28 +1,40 @@
 # .NET SDK Container Building Tools
 
-This repo contains APIs and MSBuild Tasks for generating and testing an [OCI Container](https://opencontainers.org/) from a .NET project.
+This project consists of APIs and MSBuild Tasks for generating and testing an [OCI Container](https://opencontainers.org/) from a .NET project.
 
-A basic start with the library in an existing project:
+A basic start with tooling 
+
+- for existing web project (the package is part of `Microsoft.NET.SDK.Web):
+
+```shell
+dotnet publish --os linux --arch x64 -c Release /p:PublishProfile=DefaultContainer /p:EnableSDKContainerSupport=true
```

Comment:
nit: since https://github.com/dotnet/sdk/pull/31432 is imminent maybe remove the EnableSdkContainerSupport flag?